entity-framework/core/what-is-new/ef-core-7.0/plan.md

```diff
@@ -0,0 +1,342 @@
+
+As described in the [planning process](xref:core/what-is-new/release-planning), we have gathered input from stakeholders into a plan for Entity Framework Core 7.0 (EF Core 7.0.) For brevity, EF Core 7.0 is also referred to as just EF7.
+
+> [!IMPORTANT]
+> This plan is not a commitment; it will evolve as we continue to learn throughout the release. Some things not currently planned for EF7 may get pulled in. Some things currently planned for EF7 may get punted out.
+
+## General information
+
+EF Core 7.0 is the next release after EF Core 6.0 and is currently scheduled for release in November 2022 at the same time as .NET 7. There are no plans for an EF Core 6.1 release.
+
+### Supported platforms
+
+EF7 currently targets .NET 6. This may be updated to .NET 7 as we near the release. EF7 does not target any .NET Standard version; for more information see [the future of .NET Standard](https://devblogs.microsoft.com/dotnet/the-future-of-net-standard/). EF7 will not run on .NET Framework.
+
+EF7 will align with the [.NET support policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) and will therefore **not** be a long-term support (LTS) release.
+
+### Breaking changes
+
+EF7 will contain a small number of [breaking changes](xref:core/what-is-new/ef-core-7.0/breaking-changes) as we continue to evolve both EF Core and the .NET platform. Our goal is to minimize breaking changes as much as possible.
+
+## Themes
+
+The large investments in EF7 will fall mainly under the following themes:
+
+- Highly requested features
+- NET platforms and ecosystem
+- Clear path forward from EF6
+- Performance
+
+Each of these themes is described in detail below. The high-level status of each theme can be tracked in the [EF Core biweekly updates](https://github.com/dotnet/efcore/issues/23884). Please comment on [GitHub Issue #26994](https://github.com/dotnet/efcore/issues/26994) with any feedback or suggestions.
+
+## Theme: Highly requested features
+
+As always, a major input into the [planning process](xref:core/what-is-new/release-planning) comes from [votes (👍) for features on GitHub](https://github.com/dotnet/efcore/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc). Based on these votes in addition to other factors, we plan to work on the following highly requested features for EF7.
+
+### JSON columns
+
+Tracked by [Issue #4021: Map JSON values stored in database to EF properties](https://github.com/dotnet/efcore/issues/4021)
+
+Value proposition: Save and query into JSON-based documents stored in relational database columns.
+
+This feature will introduce a common mechanism and patterns for JSON support that can be implemented by any database provider. We will work with the community to align existing implementations for [Npgsql](https://github.com/npgsql/efcore.pg) and [Pomelo MySQL](https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql), and also add support for SQL Server and SQLite.
+
+### Bulk updates
```

Comment:
Seems good to be very clear that we don't plan on doing bulk insert (also added sentence below)...

```suggestion
### Bulk updates and deletes
```